app/cypress/integration/sendingTab.spec.ts

```diff
@@ -0,0 +1,142 @@
+/// <reference types="cypress" />
+/// <reference types="@testing-library/cypress" />
+/// <reference path="../support/index.d.ts" />
+
+/* eslint-disable prefer-arrow-callback */
+describe('Sending tab', () => {
+  it('Sends a valid transaction', () => {
+    const sendAddress =
+      'addr_test1qzlaya83y9uxd2vpcfthjpefhgs2xxvc57m2epvvaj6m9r6jk3jrkjzr07qreswfvj55xvkmyayluf49gvygu5htl35q8uuu7e'
+    const sendAmount = '50'
+
+    cy.dataCy('Address text field').type(sendAddress)
+    cy.get('@MaxButton').should('be.enabled')
+
+    cy.dataCy('Send amount field')
+      .type(sendAmount)
+      .should('have.value', sendAmount)
+    cy.contains('Calculating fee...')
+
+    cy.get('@SendButton').should('be.enabled')
+
+    cy.dataCy('Send fee amount')
+      .should('be.visible')
+      .invoke('text')
+      .then(parseFloat)
+      .as('FeeAmount')
+      .should('be.gt', 0.1)
+
+    cy.get('@SendButton').click()
+    cy.contains('Transaction review').should('be.visible')
+    cy.contains(sendAddress).should('be.visible')
+    // total is the sum of previously seen fee and input amount
+    cy.dataCy('Send amount total')
+      .should('be.visible')
+      .invoke('text')
+      .then(parseFloat)
+      .should(function($totalAmount) {
+        expect($totalAmount).to.eq(this.FeeAmount + parseFloat(sendAmount))
```

Comment:
I am just thinking if we can not get strange false-positive errors in some cases with checks like there. As e.g. `0.1 +  0.2 === 0.3` returns `false` in JS, due to JS arithmetics :thinking:

Reply:
True, the edge case it's testing shouldn't happen too

Reply:
Removed, now checking whether total amount is greater than send amount / min amount